pages/home/mcp/local/code-client.mdx

```diff
@@ -0,0 +1,200 @@
+import { Steps, Tabs } from "nextra/components";
+import { SignupLink } from "@/components/Analytics";
+
+# MCP Local Client
+
+In this guide, you'll learn how to create an MCP client that can connect to a local MCP server.
+
+<Steps>
+
+### Prerequisites
+
+1. Create an <SignupLink utmParams={{
+            utm_source: "docs",
+            utm_medium: "page",
+            utm_campaign: "claude-desktop-client",
+          }} linkLocation="docs:claude-desktop-client">Arcade account</SignupLink>
+2. Get an [Arcade API key](/home/api-keys)
+
+### Install Dependencies
+
+<Tabs items={["Python", "TypeScript"]}>
+<Tabs.Tab>
+```bash
+pip install mcp
+pip install arcade-ai
+pip install arcade-google
+```
+</Tabs.Tab>
+
+<Tabs.Tab>
+```bash
+npm install @modelcontextprotocol/sdk
```

Comment:
if they wanted to write a typescript client for stdio, then they would still need to pip install arcade-ai which feels odd